Assets/Tests/Console/ConsoleTest.cs

```diff
@@ -62,6 +64,18 @@ public void SignOut()
 		AddConsoleLine(string.Format("Sign Out {0}.", isSignedIn ? "Successful" : "Failed"));
 	}
 
+	public void GetUsersById() {
+		var ids = ParseIds(userIdsField.text);
```

Comment:
The initial goal of the console was to give people an easy way in but I've never released it officially (at least not outside of the repo), but to keep things consistent, it would be great to have `AddConsoleLine("Get Users by ID. Click to see source.")`